run_project_tests.py

```diff
@@ -356,6 +356,13 @@ def have_d_compiler():
         return True
     return False
 
+def have_objc_compiler():
+    if shutil.which("objc") and shutil.which("objcpp"):
+        return True
```

Comment:
Any idea to check that then?

I have normal gcc installed of course but not the gobjc* packages, and the tests fail for me currently.